docs/csharp/language-reference/keywords/is.md

```diff
@@ -101,16 +101,22 @@ The following example shows a comparison of `null` checks:
 
 ### var pattern
 
-The `var` pattern is a catch-all for any type or value. The value of *expr* is always assigned to a local variable the same type as the compile time type of *expr*. The result of the `is` expression is always `true`. Its syntax is:
+A pattern match with the `var` pattern always succeeds. Its syntax is:
 
 ```csharp
    expr is var varname
 ```
 
-The following example uses the var pattern to assign an expression to a variable named `obj`. It then displays the value and the type of `obj`.
+where the value of *expr* is always assigned to a local variable named *varname*. *varname* is a variable of the same type as the compile-time type of *expr*. 
+
+If *expr* evaluates to `null`, the `is` expression produces `true` and assigns `null` to *varname*. 
```

Comment:
Is it worth adding a clarifying comment here?  In most situations `is` produces `false` for `null`:

```suggestion
If *expr* evaluates to `null`, the `is` expression produces `true` and assigns `null` to *varname*.  The var pattern is one of the few uses of `is` that produces `true` for a `null` value.
```

Reply:
I don't think it's particularly useful. Side question, are there other uses of var that does produce true for null?

Reply:
@Thraka Nope, `thing is var newVariableName` always succeeds, and always returns true.

But, `thing is string message` returns `false` if `thing` is `null`.